daphne_worker/src/dap.rs

```diff
@@ -548,17 +548,24 @@ where
         // Decide which reports to reject early. A report will be rejected if has been processed
         // but not collected or if it has not been proceessed but pertains to a batch that was
         // previously collected.
+        let current_time = self.get_current_time();
         let mut early_fails = HashMap::new();
         for (bucket, collected) in agg_store_request_bucket
             .iter()
             .zip(agg_store_responses.into_iter())
         {
+            let min_time = self.least_valid_report_time(current_time);
```

Comment:
We can move `min_time` and `max_time` out of the for loop, since their value does not change.